templates/etc/admin/admin.conf.erb

```diff
@@ -0,0 +1,87 @@
+# The pulp server configuration
```

Comment:
It should be all or nothing though, if you want admin moved to the top level, everything else should be too